central/src/containers/RouteContainer.tsx

```diff
@@ -299,19 +334,42 @@ export const RouteContainer = ({
     const newQuestionTemplate = { ...rest, id: uuidv4(), title: `Clone of ${question.title}`};
     const result = await createQuestionTemplate(newQuestionTemplate);
     if (result) {
-      getAllGameTemplates(nextToken);
+      listQuerySettings.nextToken = nextToken;
+      getAllGameTemplates(listQuerySettings);
       setAlert({ message: 'Question cloned.', type: 'success' });
     }
     return result
   }
 
```

Comment:
this updates the `numGameTemplates` and `numQuestionTemplates` fields in the respective `questionTemplate` and `gameTemplate`. For use in sorting by question/game count